### PR TITLE
FE-162: Display geodesic lines instead in geo visualization

### PR DESCRIPTION
**Pull request by ajurdak**
### Scope & Purpose

This PR solves one of two geo visualization issues. With the added package "leaflet.geodesic" lines are now displayed geodesic and not straight.

For example `RETURN GEO_Polygon([[47.56, -122.33], [-33.94, 18.39], [39.92, 116.39], [48.2084114, 16.3708958], [47.56, -122.33]])` in the query editor in the WebUI the result looks now like this:
image

- [x] :hankey: Bugfix

### Checklist

- [x] Manually tested
- [x] :book: CHANGELOG entry made
- [ ] Backports
  - [x] Backport for 3.11: https://github.com/arangodb/arangodb/pull/18830
  - [ ] Backport for 3.10: ToDo

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/BTS-777

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js:1]: why do we need to have this file in our source code? Can't we use the [npm package](https://www.npmjs.com/package/leaflet.geodesic) for this?

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js [hunk 1]
```diff
@@ -2211,12 +2211,7 @@
                 }
               } else if (geometry.type === 'Polygon' || geometry.type === 'LineString' || geometry.type === 'MultiLineString' || geometry.type === 'MultiPolygon') {
                 try {
-                  geojson = new L.GeoJSON(geometry, {
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js:2214]: is L.GeoJSON inbuilt, or was it another leaflet package? If it was and it's unused now, maybe we can remove it?

**Review by palashkaria — CHANGES_REQUESTED** [review]

File: js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js:1]: let's try using package.json for this new dependency @ajurdak

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js (hunk 1 above)
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js:2214]: I guess that specific package can still remain, but we can move others (like leaflet itself) to npm? Should work, but will need to try

File: js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js
Comment (ajurdak) [reply js/apps/system/_admin/aardvark/APP/frontend/js/lib/leaflet.geodesic.js:1]: With eb2d1c963d42a01da8de69b8190d784775f01cf5

- package.json is used for the dependencies
- Unnecessary files are deleted

**Review by palashkaria — APPROVED** [review]
LGTM

**Comment by palashkaria** [comment]
@ajurdak I just noticed, there are a lot of breaking changes between 1.3.3 to 1.9.3. Have you verified we are not using any of them?
https://github.com/Leaflet/Leaflet/blob/main/CHANGELOG.md

**Comment by Simran-B** [comment]
Looks like the example had longitude and latitude in the wrong order. This gives me the same result as in the screenshot:

`RETURN GEO_POLYGON([[-122.33, 47.56], [18.39, -33.94], [116.39, 39.92], [16.3708958, 48.2084114], [-122.33, 47.56]])`

**Pull request merged**